### PR TITLE
Young Xenos Buff

### PR DESCRIPTION
**Pull request by Isy232**
## About The Pull Request

Buffs Young Xenos.

Buffs young, mature and elder T1's to be closer to ancient.
Buffs young and mature T2's to be closer to elder.
Buffs young T3's to be closer to mature.

## Why It's Good For The Game

This should make the game more forgiving to xenos and make kills at the start of the game not decide the next 30 minutes of the game.

## Changelog
:cl:
balance: Buffs younger xenos
/:cl:

### CODE REVIEW

**Comment by tgstation-server** [comment]
You currently have a negative Fix/Feature pull request delta of -3. Maintainers may close this PR at will. Fixing issues or improving the codebase will improve this score.

**Comment by Kuro-ai** [comment]
I'm against these changes,not only did Isy not talk in balance at all about this,it's buffing xenos to deal more damage than they did pre damage deescalation,xenos really don't need damage buffs right now

**Comment by Pariah919** [comment]
These files are old and this feels a lot like powercreep personally to make the scaling even more near 

If you want to nerf marine rushing then make a system where yung t1s and t2s get mature on door opening or DS landing.

**Comment by Kuro-ai** [comment]
Aside from that,on my marine armor pr most armor sets were nerfed aside from heavy armor and b18 further making these damage buffs a mistake

**Comment by Isy232** [comment]
Only damage buffs to young xenos so they are not as ineffective as older ones.

The goal of this PR is to make it so that the game isn't over if the first xenos die.

**Comment by Isy232** [comment]
If you think about it by the time the marines land all the T1's are ancient, T2's elder and T3's mature.
In that situation this PR would not buff xenos at all.

**Comment by Kuro-ai** [comment]
a damage buff is still a damage buff,if it wasn't that important you wouldn't be buffing it at all,this is power creep at it's finest

**Comment by Isy232** [comment]
No, this isnt a buff to xenos damage when xenos and marines would first encounter eachother.

What this allows is xenos to take losses better so it isnt game over if the first few xenos die.

**Comment by Kuro-ai** [comment]
this is powercreep,it's upping the floor of xeno power and leaving marines in the dust when A)marine armor (aside from heavy and b18) got nerfed and current xenos can dispatch a light/medium armor marine without much effort B)xenos got their damage deescalated for a reason,reducing rocket tag,and you're just undoing all of that

**Comment by Isy232** [comment]
I am not undoing anything, the numbers for when xenos and marines meet are THE EXACT SAME. 

Its only once the xenos are loosing that this will be noticeable.

File: code/modules/mob/living/carbon/xenomorph/castes/boiler/castedatum_boiler.dm [hunk 1]
```diff
@@ -38,10 +38,10 @@
 	armor = list("melee" = 30, "bullet" = 30, "laser" = 30, "energy" = 30, "bomb" = XENO_BOMB_RESIST_0, "bio" = 30, "rad" = 30, "fire" = 30, "acid" = 30)
 
 	// *** Ranged Attack *** //
-	spit_delay = 4 SECONDS
+	spit_delay = 2.5 SECONDS
 
 	// *** Boiler Abilities *** //
-	bomb_strength = 1 //Multiplier to the effectiveness of the boiler glob. Improves by 0.5 per upgrade
+	bomb_strength = 1.5 //Multiplier to the effectiveness of the boiler glob. Improves by 0.5 per upgrade
```
Comment (Fikou) [inline code/modules/mob/living/carbon/xenomorph/castes/boiler/castedatum_boiler.dm]: "Improves by 0.5 per upgrade"

File: code/modules/mob/living/carbon/xenomorph/castes/boiler/castedatum_boiler.dm [hunk 2]
```diff
@@ -88,10 +88,10 @@
 	armor = list("melee" = 35, "bullet" = 35, "laser" = 35, "energy" = 35, "bomb" = XENO_BOMB_RESIST_0, "bio" = 35, "rad" = 35, "fire" = 35, "acid" = 35)
 
 	// *** Ranged Attack *** //
-	spit_delay = 3 SECONDS
+	spit_delay = 2.5 SECONDS
 
 	// *** Boiler Abilities *** //
-	bomb_strength = 1.5
+	bomb_strength = 1.75
```
Comment (Fikou) [inline code/modules/mob/living/carbon/xenomorph/castes/boiler/castedatum_boiler.dm]: but it improved by 0.25

**Comment by Rohesie** [comment]
Closing due to staleness.

**Pull request closed without merging**